jsonschema-validation.xml

```diff
@@ -1043,8 +1043,14 @@
                     is going to be removed in the future.
                 </t>
                 <t>
-                    A root schema containing "deprecated" with a value of true indicates the entire
-                    root schema MAY be removed in the future.
+                    A root schema containing "deprecated" with a value of true indicates that
+                    the entire resource beng described MAY be removed in the future.
+                </t>
+                <t>
+                    When the "deprecated" keyword is used to describe an array, it's meaning will
+                    vary depending on how "items" is used. If "items" is a single schema, it will
+                    deprecate the contents of an array, but if "items" is an array of schemas it
+                    can be used to deprecate positional schemas.
```

Comment:
See, I understand this, but in the case where one wants to signal deprecation of a positional array element, wouldn't it make more sense to have `deprecated` set on the associated schema inside `items`?

Reply:
Isn't that what this says? Can you try the suggestion feature again to improve the words? Words are hard.

Reply:
No, it says when it's applicable to an array, while if you want it to mean what I've said, you mean, "when it's applicable to an item in an array.

Currenly, it reads as if you define `deprecated` in the same object as `items`, rather than in a subschema defined inside `items`. See the difference?

If your intent is for it to be defined in the same object as `items` where `items` is an array of subschemas, how would you know to which subschema it applies positionally?

Reply:
@Relequestual I took another stab at fixing this.

> but if "items" is an array of schemas it should be defined in the sub-schema targeted for deprecation.

does that make more sense? The previous sentance is saying:

´``
"items": {
   "type": "string",
   "deprecated": true
}
```

based on your understanding you think im suggesting it suddenly jumps _up_ a keyword somehow?

No it would look like this:

´``
"items": [{
   "type": "string",
}, {
   "type": "string",
   "deprecated": true
}],
```

Now the 2nd thing is going away.

Reply:
I'll start a new review for this as this is now outdated...